test/k8sT/Updates.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var _ = Describe("K8sValidatedUpdates", func() {
+var _ = Describe("DisabledK8sValidatedUpdates", func() {
```

Comment:
Can you add a comment to #3442 to undo this?

Reply:
Done

Reply:
There is also 
```
It("Updating Cilium stable to master", func() {
		// GH-3442 Re-enable once stable
		return
````
@aanm told me this was not working reliably and so he had to add this code. #3442 needs to undo both this `return` code as well as the `Disabled` keyword to enable this test correctly.